shared/markdown/__test__/parser.test.js

```diff
@@ -9,7 +9,7 @@ describe('Markdown parser', () => {
   })
 
   it('parses invalid emoji fragments correctly', () => {
-    const ast = parser.parse('one::\n::two\n:three?::\n::four:\n::')
+    const ast = parser.parse('one::\n::two\n:three?::\n::four:\n:: :+1:')
```

Comment:
:sheep: Could you please put the `:+1` in a different test case or rename the test case? This one was intended for invalid fragments. :neckbeard: